apps/api/Dockerfile

```
@@ -0,0 +1,87 @@
+# -------------------------------
+# Builder stage
+# -------------------------------
+FROM python:3.13-slim AS builder
+
+# Enable fail-fast: exit immediately on error, fail on pipe errors
+SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# System dependencies for building Python wheels
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)